lib/config.js

```diff
@@ -11,6 +11,7 @@ var fs = require('fs');
 var open = require('fs').openSync;
 var path = require('path');
 var sendMetrics = require('./metrics');
+var sendExpressMetrics = require('./express-metrics');
```

Comment:
I'll use `express-records` and `sendExpressRecords()`, as the code is sending more than one record.